### DISCUSSION TITLE
Can put this area to the Center? ALL Pages。 It will look better if you put it in the middle.

### DISCUSSION THREAD

**Discussion by yangrunkang** [withastro/starlight#1772] [Feature Requests]
### What version of `starlight` are you using?

0.21.1

### What is your idea?

It will look better if you put it in the middle.

![image](https://github.com/withastro/starlight/assets/46257933/c2a36086-b90e-4122-a6ca-3693c87d6aee)

### Why is this feature necessary?

It will look better if you put it in the middle.

### Participation

- [ ] I am willing to submit a pull request for this feature.

**Comment by HiDeoo** [comment]
Thanks for the feedback.

> It will look better if you put it in the middle.

That's definitely up to personal preference, and not something we'd like to change for now. Altho, such behavior can be achieved [using custom CSS](https://starlight.astro.build/guides/css-and-tailwind/#custom-css-styles) with the following code:

```css
/* src/styles/custom.css */

.main-pane {
  --sl-content-margin-inline: auto;
}
```
